modules/shortcodes/flickr.php

```diff
@@ -189,19 +150,98 @@ function flickr_shortcode_handler( $atts ) {
 /**
  * Return HTML markup for a Flickr embed.
  *
- * @param array $atts Shortcode attributes.
+ * @param array  $atts Shortcode attributes.
+ * @param string $id Video ID.
+ * @param string $video_param video param of the shortcode.
  */
-function flickr_shortcode_video_markup( $atts ) {
-	$atts = array_map( 'esc_attr', $atts );
+function flickr_shortcode_video_markup( $atts, $id, $video_param ) {
```

Comment:
It may be interesting to test that new function, as I can imagine that og:url tags or other things may change in the future and impact us.

Reply:
I added more specific unit tests for this function, thanks for the suggestion.